server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
@tombuildsstuff @matteofigus Just seen this, still want to sort this out over a 🍺 ?